main.tf

```diff
@@ -367,6 +367,22 @@ resource "aws_codebuild_project" "default" {
     }
   }
 
+  dynamic "secondary_sources" {
+    for_each = var.secondary_sources ? [""] : []
```

Comment:
you can just do `for_each = var.secondary_sources`

Reply:
Done